bokeh/resources.py

```diff
@@ -42,11 +42,11 @@ def _get_cdn_urls(version=None, minified=True):
     base_url = _cdn_base_url()
     rel_container = 'bokeh/release'
     dev_container = 'bokeh/dev'
-    container = rel_container if dev != 'dev' else dev_container
+    container = rel_container if not dev.startswith(('dev', 'rc')) else dev_container
```

Comment:
jaja... `-(-1) = 1` :stuck_out_tongue: but fair enough, I will change it...

Just as a comment, there is a reason I write it in that way... for western people, you read from left to right AND from top to bottom, so... as I first defined `rel_container` and then `dev_container` is is natural for me to make the construction with `rel_container` as the default condition... so, to be consistent with my idea, I will change this one to:
```python
     dev_container = 'bokeh/dev'
     rel_container = 'bokeh/release'
     container = dev_container if dev.startswith(('dev', 'rc')) else rel_container
```

Yes, I know this is subtle difference, but I firmly believe this kind of details are important from the readability point of view... OK, no more "nonsense linguistic", jaja...